### PR TITLE
nsnake: add page

### PR DESCRIPTION
**Pull request by sahilister**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/nsnake.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# nsnake
+
+> Snake game in terminal.
```
Comment (sbrl) [inline pages/linux/nsnake.md:3]: Perhaps this will read better?
```suggestion
> Snake game in the terminal.
```

**Review by sbrl — COMMENTED** [review]
Thanks for the new page! I've left some comments below for you to review.

Hrm, haven't I reviewed a page similar to this recently :thinking:

File: pages/linux/nsnake.md (hunk 1 above)
Comment (sahilister) [reply pages/linux/nsnake.md]: @sbrl I recently made pull request for `snake4` #4895 .

Most of the arguments were similar, so I took that as a boilerplate.

**Comment by sahilister** [comment]
@sbrl I recently made pull request for `snake4` #4895  .

Most of the arguments were similar, so I took that as a boilerplate.

**Review by navarroaxel — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]

**Pull request merged**